index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
- [ ] For this milestone you are expected to create a header for mobile devices. However, the width of the page is fixed `.header { width: 144rem;} `, I suggest to delete the fixed width and  develop the mobile header: 

https://user-images.githubusercontent.com/77038610/153429881-fbe7a291-b2d0-4dbf-86c3-2b1ae0f2e89f.mp4

- [ ] Please make sure that the mobile heading looks exactly like on Figma. 

- [ ] Please add humburger button on the right side as provided on Figma:

![image](https://user-images.githubusercontent.com/77038610/153430727-0adb2de6-2800-4ebd-bf50-bf727f299d13.png)